src/student.js

```diff
@@ -0,0 +1,16 @@
+class Student {
+  constructor(id, studentName, gitHub, email) {
+    this.id =
+      id +
+      '_' +
+      studentName.split(' ')[0][0] +
+      '_' +
+      studentName.split(' ')[1][0]
```

Comment:
this is interesting, I didn't think to use this

Reply:
i don't know what this does but It looks interesting and I loved to know